kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -219,10 +219,10 @@ module.controller('grImageMetadataCtrl', [
 
       ctrl.additionalMetadata = Object.fromEntries(
         Object.entries(ctrl.singleImage.data.aliases)
-          .map(([key, val]) => {
+          .map(([key, value]) => {
             let fieldAlias = ctrl.fieldAliases.find(_ => _.alias === key);
             if (fieldAlias && fieldAlias.displayInAdditionalMetadata === true) {
-              return [fieldAlias.label, val];
+              return [fieldAlias.label, {value,elasticsearchPath: fieldAlias.elasticsearchPath}];
```

Comment:
you could try this (complete guess, no idea what angular will try to do with it)
```suggestion
              return [fieldAlias.label, { value, elasticsearchPath: '"' + fieldAlias.elasticsearchPath + '"' }];
```

Reply:
Thanks! It makes it much better, but still, under certain scenarios, a good search flashes and as if aftyer a reload ui-router (?) messes things up…